rusk-recovery/src/state.rs

```diff
@@ -4,15 +4,25 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
+use crate::provisioners::PROVISIONERS;
 use crate::theme::Theme;
+
 use microkelvin::{Backend, BackendCtor, DiskBackend, PersistError};
+use phoenix_core::Note;
+use rand::rngs::StdRng;
+use rand::SeedableRng;
 use rusk_abi;
 use rusk_vm::{Contract, NetworkState, NetworkStateId};
-use stake_contract::StakeContract;
+use stake_contract::{Stake, StakeContract, MINIMUM_STAKE};
 use std::{fs, io};
 use tracing::info;
 use transfer_contract::TransferContract;
 
+/// Initial amount of the note inserted in the state.
+const GENESIS_DUSK: u64 = 1_000_000_000; // 1000 Dusk.
+/// The number of blocks after which the genesis stake expires.
+const GENESIS_EXPIRATION: u64 = 1_000_000;
```

Comment:
I think we have no more expiration on stakes. But if this is just to comply with API I'm fine with it